pkg/view/component/terminal_test.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright (c) 2019 the Octant contributors. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package component
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerminal_Marshall(t *testing.T) {
+	details := TerminalDetails{
+		Container: "container-id",
+		Command:   "/bin/bash",
+		UUID:      "0000-0000-0000-0000-0000",
+		Active:    false,
+	}
+	input := NewTerminal("default", "term-test", details)
+	actual, err := json.Marshal(input)
+	assert.NoError(t, err)
+
+	var expected = `
```

Comment:
If you put this in `testdata/whatever`, you could load it from the filesystem and not have to deal with weird formatting in a string.